public/styles/components/_chart.scss

```diff
@@ -0,0 +1,33 @@
+.chart-wrap {
+  position: relative;
+}
+
+$loadingLabelWidth: 100px;
+$loadingLabelHeight: 30px;
+
+.chart-wrap:after {
+  content: 'Loading\2026'; // Ellipsis
+  display: none;
+  text-align: center;
+  line-height: $loadingLabelHeight;
+  width: $loadingLabelWidth;
+  position: absolute;
+  top: calc(50% - #{$loadingLabelHeight} / 2);
```

Comment:
Does it make sense to try and parameterise this? Would this make it more clear on where these values come from?